### PR TITLE
Add check and transfer that validates using merkle validator

### PR DESCRIPTION
**Pull request by sunwrobert**

### CODE REVIEW

File: src/schemas/ERC1155/index.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import * as Web3 from 'web3';
-
 import {
   FunctionInputKind,
   FunctionOutputKind,
   Schema,
-  StateMutability,
+  StateMutability
```
Comment (alexanderatallah) [inline src/schemas/ERC1155/index.ts]: nit: can we add an eslint rule for this? my editor keeps going back and forth and is very confused

**Review by alexanderatallah — APPROVED** [review]
approved with nit

File: package.json [hunk 2]
```diff
@@ -51,6 +51,6 @@
     "bignumber.js": "^6.0.0",
     "typed-promisify": "^0.4.0",
     "web3-provider-engine": "^13.6.5",
-    "wyvern-js": "git+https://github.com/ProjectOpenSea/wyvern-js.git#v3.3.2"
+    "wyvern-js": "git+https://github.com/ProjectOpenSea/wyvern-js.git#adb0b5ceb8616ce1414096a27c0a8a953120c0aa"
```
Comment (alexanderatallah) [inline package.json]: can we use version number here, and also update this package to version 0.15?

**Review by alexanderatallah — CHANGES_REQUESTED** [review]
added a version comment

File: src/schemas/ERC1155/index.ts (hunk 1 above)
Comment (sunwrobert) [reply src/schemas/ERC1155/index.ts]: you using prettier?

Comment (sunwrobert) [reply src/schemas/ERC1155/index.ts]: Just going to add the comma back, don't think this repo has normal eslint setup so it's going to be messy until we actually add that in, but out of scope

**Comment by sunwrobert** [comment]
I set the correct version now. Merging.

File: package.json (hunk 2 above)
Comment (sunwrobert) [reply package.json]: Will update to 0.15

**Pull request merged**